### DISCUSSION TITLE
Is there any plan to develop a tool similar to typedoc?

### DISCUSSION THREAD

**Discussion by jeffwcx** [oxc-project/oxc#7878] [Ideas]
Is there any plan to develop a tool similar to [typedoc](https://github.com/TypeStrong/typedoc)? I believe this is a very important part of the infrastructure.
However, [typedoc](https://github.com/TypeStrong/typedoc) is highly associated with the typechecker, and I'm not sure if the existing infrastructure of oxc can accomplish this.

**Comment by Boshen** [comment]
Not planned.
